polaris.shopify.com/src/components/StatusBanner/StatusBanner.tsx

```diff
@@ -10,7 +12,7 @@ function StatusBanner({status: {value, message}}: Props) {
   return (
     <div className={styles.StatusBanner} data-value={value.toLowerCase()}>
       <h2>{uppercaseFirst(value)}</h2>
-      <p>{message}</p>
+      <Markdown text={message} />
```

Comment:
any reason we wouldn't want to support markdown here in the future?

Reply:
Nope.